.github/workflows/pr.yaml

```diff
@@ -0,0 +1,64 @@
+name: PR Build
+
+on:
+  pull_request:
+   types:
+    - opened
+    - synchronize
+    - reopened
+
+defaults:
+  run:
+    shell: bash
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Set up Helm
+      uses: azure/setup-helm@v1
+      with:
+        version: v3.8.1
+
+    - uses: actions/setup-python@v2
+      with:
+        python-version: "3.10"
+
+    - name: Set up chart-testing
+      uses: helm/chart-testing-action@v2.2.1
+
+    - name: Run chart-testing (list-changed)
+      id: list-changed
+      run: |
+        changed=$(ct list-changed --target-branch ${{ github.event.repository.default_branch }})
+        if [[ -n "$changed" ]]; then
+          echo "::set-output name=changed::true"
+        fi
+
+    - name: Run chart-testing (lint)
+      run: |
+        ct lint \
+          --target-branch ${{ github.event.repository.default_branch }} \
+          --lint-conf chart-testing/lintconf.yaml \
+          --excluded-charts charts/common
+      if: steps.list-changed.outputs.changed == 'true'
+
+    - name: Create kind cluster
+      uses: helm/kind-action@v1.2.0
+      with:
+        node_image: kindest/node:v1.21.10
+      # if: steps.list-changed.outputs.changed == 'true'
+
+    - name: Run chart-testing (install)
+      run: ct install --charts charts/polkadot-introspector --charts charts/substrate-faucet --charts charts/substrate-telemetry
```

Comment:
Would it be possible to automatically include all charts in the `charts/` directory ?

Reply:
Last time I looked there was an `--all` flag although the behaviour is dependent on changed versions unlike `--charts` which will run against any chart specified, irrespective of changes or not.

Reply:
Yep, I'm just testing all charts individually to catch the errors early